src/libstd/sys/windows/c.rs

```diff
@@ -1175,6 +1160,9 @@ extern "system" {
                   writefds: *mut fd_set,
                   exceptfds: *mut fd_set,
                   timeout: *const timeval) -> c_int;
+
+    #[link_name = "SystemFunction036"]
```

Comment:
Oh of course, `SystemFunction036` 😨

Reply:
`um\NTSecAPI.h`
`#define RtlGenRandom                    SystemFunction036`

Reply:
Could you add a comment about why its used.